Marko doc (#621)
Macrometacorp/docs: docs/best-practice/use-o-indexes-for-collect-operation.md

---
sidebar_position: 50
title: Use of indexes for COLLECT operation
---

> If there is a `COLLECT` operation in the query, the records with similar attribute values are grouped.  Persistent index on the attribute value on which `COLLECT` operation is performed helps to optimize the query. In the following example, the persistent index on the `country` attribute will help to optimize the query.

Reviewer: ```suggestion
If there is a `COLLECT` operation in the query, then the records with similar attribute values are grouped.  Persistent indexes on the attribute value on which `COLLECT` operation is performed helps to optimize the query. In the following example, the persistent index on the `country` attribute helps to optimize the query.
```
